### PR TITLE
Copy back missing docs and build scripts

### PR DESCRIPTION
**Pull request by carlosms**
Part of #78.

Instead of overriding `build` I left `superset-build` as a new target. Without `COMMANDS` defined the go `build` does nothing, and this way `make build` does not complain about ignored commands.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+# Prints unified diff of local superset  and upstream
+.PHONY: diff
+diff: superset-remote-add
+	git diff-tree -p $(SUPERSET_REMOTE)/$(SUPERSET_VERSION) HEAD:$(SUPERSET_DIR)/
+
+# Merge remote superset into SUPERSET_DIR as squashed commit
+.PHONY: merge
+merge: superset-remote-add
+	git merge --squash -s subtree --no-commit remotes/$(SUPERSET_REMOTE)/$(SUPERSET_VERSION)
```
Comment (smacker) [inline Makefile]: could you please remove this? we are using `git subtree` now.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+# superset upstream configuration
+SUPERSET_REPO = https://github.com/apache/incubator-superset.git
+SUPERSET_VERSION = release--0.32
+SUPERSET_REMOTE = superset
+# directory to sync superset upstream with
+SUPERSET_DIR = superset
+# directory with custom code to copy into SUPERSET_DIR
+PATCH_SOURCE_DIR = srcd
+# name of the superset docker image to build
+SUPERSET_IMAGE_NAME = srcd/superset
```
Comment (smacker) [inline Makefile]: maybe it should be `srcd/sourced-ui` (I don't know)

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,94 @@
+# Package configuration
+PROJECT = sandbox-ce
```
Comment (smacker) [inline Makefile]: sourced-ui

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+# superset upstream configuration
+SUPERSET_REPO = https://github.com/apache/incubator-superset.git
+SUPERSET_VERSION = release--0.32
```
Comment (smacker) [inline Makefile]: `0.32.0rc2`

File: Makefile (hunk 1 above)
Comment (carlosms) [reply Makefile]: do `diff` and `diff-stat` need any change?

File: README.md [hunk 5]
```diff
@@ -1 +1,58 @@
...
+
+## Contents
+
+- [Development](#development)
+
+## Development
+
+### Setup local environment
+
+Run dependencies using docker-compose:
```
Comment (smacker) [inline README.md]: hm... but the repository doesn't have docker-compose.yml anymore

File: Makefile (hunk 2 above)
Comment (carlosms) [reply Makefile]: yes, keeping the same name as the repo sounds better.

File: Makefile (hunk 4 above)
Comment (carlosms) [reply Makefile]: this was changed during the project rename? Because I copied the values from `superset-compose` master.

File: Makefile (hunk 1 above)
Comment (smacker) [reply Makefile]: nope. There are no commands for diff in `git subtree`.

File: Makefile (hunk 4 above)
Comment (smacker) [reply Makefile]: yep: https://gist.github.com/smacker/f0a317d5e618eedbc5b77ba36cad1f0c#file-move_superset-txt-L13
They actually point on the same commit most probably but still

**Comment by carlosms** [comment]
I fixed all comments on 0f39b5ba.

File: Makefile (hunk 2 above)
Comment (carlosms) [reply Makefile]: I also created the PR to change it in the compose file https://github.com/src-d/sourced-ce/pull/5

**Comment by smacker** [comment]
ok. I need it. Merging.

**Pull request merged**